config/custom-environment-variables.json

```diff
@@ -14,7 +15,8 @@
   },
   "api": {
     "url": "TRIBUNALS_CASE_API_URL",
-    "uploadEvidenceUrl": "UPLOAD_EVIDENCE_URL"
+    "uploadEvidenceUrl": "UPLOAD_EVIDENCE_URL",
+    "draftUrl": "DRAFT_API_URL"
```

Comment:
DRAFT_API_URL is not configured in the main.tf 
The FE in this PR is not using the endpoint yet, but since you are declaring it we should also configure it in the main.tf, I think